packages/client/src/utils.ts

```diff
@@ -57,8 +57,21 @@ export const decodeToken = (token: string): IDToken => {
 
 export const nowRoundToSec = () => Math.floor(Date.now() / 1000);
 
-export const encodeBase64 = (input: string) => btoa(input);
-export const decodeBase64 = (input: string) => atob(input);
+const replaceNonUrlSafeCharactersInBase64 = (rawString: string) => {
```

Comment:
https://github.com/RGBboy/urlsafe-base64/blob/master/lib/urlsafe-base64.js looks pretty handy

Reply:
or we could fork the usage of this package. look pretty elegant to me 

```js
const UrlSafeBase64 = {
 encode: () => {},
 decode: () => {},
}
```